### ISSUE TITLE
Interactive Bill of Materials 

### ISSUE THREAD

**Issue by PowerfulTitan** [pycubed/hardware#10]
Hello!

I am building some battery boards for my college's cubesat team. The links to the interactive bill of materials appear to be broken; I was wondering if this was available somewhere else or if there was a way I could view it?

**Comment by maholli** [comment]
Hi yeah the website has been having trouble and I haven't had the chance to fix it. 

The github repos still have all the info you're needing, though. If you're looking for the battery board (v01b) BOM, [download this file](https://github.com/pycubed/hardware/blob/master/batteryboard-v01b/bom/index.html) (if you don't want to clone the repo) and open it with your browser. 

Let me know if it doesn't work.

**Comment by PowerfulTitan** [comment]
We have all the components ready for building (bought in previous years); I was more so interested in determining what went where, like the interactive BOM showed. The battery board has some labels (R1, R2, ..., C1, C2, ...), and right now, I am planning on just placing components based on that, which I think should be fine.

**Comment by maholli** [comment]
yep that link in the previous message is the interactive BOM html file. Download the file and open it in your browser to see where things go.

**Comment by PowerfulTitan** [comment]
It worked, thank you!
